### ISSUE TITLE
[Request]Donation Button at Homepage

### ISSUE THREAD

**Issue by blackcrack** [QuiteRSS/quiterss#1217]
Adding a Donationbutton at the Homepage with a 
Donation Page where be more informations for Supporting.

...

best regards
blacky

**Comment by Funcy-dcm** [comment]
fix #1218

**Comment by blackcrack** [comment]
@Funcy-dcm 


ehh.. and the Homepage ?
https://quiterss.org/

```
<div class='button'><a href='https://quiterss.org/en/donations' title='Donation, Support you're Newsreader' target='_blank'>Donations</a></div>
```
Create a css class for a normal Button..

best regards
Blacky
